content/en/blog/_posts/2023-11-02-mid-cycle-1.29.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Kubernetes Removals, Deprecations, and Major Changes in Kubernetes 1.29'
+date: 2023-11-09
+slug: kubernetes-1-29-release-announcement
+---
+
+**Authors:** Carol Valencia, Kristin Martin, Abigail McCarthy, James Quigley, Hosam Kamel
```

Comment:
Typically we add company attributions, but that's not officially required.

Reply:
We actually removed the affiliations, per a comment by Priyanka on the HackMD draft.

Reply:
OK, but: why? It would be good to know why we're not listing affiliations.

Reply:
So looking at the comments in the hackmd, I think that Priyanka comment referred to the original draft of the blog that listed "the release team" as the authors instead of the specific members of the comms team. I also looked back at the past 4 mid-cycle blogs, half include company affiliations and half do not. 

I'm fine defaulting to the usual way blogs do this.